src/Evolution/Systems/Burgers/BoundaryCorrections/Rusanov.hpp

```diff
@@ -0,0 +1,108 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <memory>
+#include <optional>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/Burgers/BoundaryCorrections/BoundaryCorrection.hpp"
+#include "Evolution/Systems/Burgers/Tags.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/Formulation.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Burgers::BoundaryCorrections {
+/*!
+ * \brief A Rusanov/local Lax-Friedrichs Riemann solver
+ *
+ * Let \f$U\f$ be the evolved variable, \f$F^i\f$ the flux, and \f$n_i\f$ be
+ * the outward directed unit normal to the interface. Denoting \f$F := n_i
+ * F^i\f$, the %Rusanov boundary correction is
+ *
+ * \f{align*}
+ * G_\text{Rusanov} = \frac{F_\text{int} - F_\text{ext}}{2} -
+ * \frac{\text{max}\left(|U_\text{int}|, U_\text{ext}|\right)}{2}
+ * \left(U_\text{ext} - U_\text{int}\right),
+ * \f}
+ *
+ * where "int" and "ext" stand for interior and exterior. The minus sign in
+ * front of the \f$F_{\text{ext}}\f$ is necessary because the outward directed
+ * normal of the neighboring element has the opposite sign, i.e.
+ * \f$n_i^{\text{ext}}=-n_i^{\text{int}}\f$.
+ *
+ * \note In the strong form the `dg_boundary_correction` function must return
```

Comment:
1. `dg_boundary_correction` -> `dg_boundary_terms` if you are talking about the member function of this class
2. "must return" -> "returns" so the statement is unambiguously true, instead of a statement of a requirement that may or may not be satisfied.